packages/metals-vscode/src/testExplorer/types.ts

```diff
@@ -135,34 +139,50 @@ export interface TestSuiteRun {
   testCases: vscode.TestItem[];
 }
 
-export type MetalsTestItemKind = "project" | "package" | "suite" | "testcase";
+export type RunnableMetalsTestItemKind = "suite" | "testcase";
+
+export type MetalsTestItemKind =
+  | "workspaceFolder"
+  | "project"
```

Comment:
I wonder if `module` wouldn't be a better name instead of project

Reply:
Agreed, "project" is confusing in this context. If anything a workspace folder should be named "project".